### ISSUE TITLE
[CoE Starter Kit - BUG] Upgrade action failed: Solution dependencies exist, cannot uninstall. ObjectName: Office 365 Management API

### ISSUE THREAD

**Issue by dcurt1977** [microsoft/coe-starter-kit#6270]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

After successfully importing the Core Components ver 4.13 to correct the issues with queries in the production Power BI dashboard, i got the below error message. I have successfully set up the Office 365 Management API prior to importing the latest version.

![image](https://github.com/microsoft/coe-starter-kit/assets/123478097/db9c16e1-417a-4b2f-b5bf-8284bf684f12)

### Expected Behavior

Center of Excellence - Core Components version 4.13 to import successfully no errors.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.13

### What app or flow are you having the issue with?

None

### What method are you using to get inventory and telemetry?

None

### Steps To Reproduce

1. Import Center of Excellence - Core Components version 4.13
2. Waited for the import End Time to complete with result status

**Comment by dcurt1977** [comment]
I am not able to delete the Office 365 Management API custom connector from the default environment as suggested by others. 

![image](https://github.com/microsoft/coe-starter-kit/assets/123478097/0ddbda6a-175f-4679-a8ce-6b80f401e8ee)

**Comment by dcurt1977** [comment]
Do I also need to delete the New Audit Log Solution and New Audit Log CC solutions?

**Comment by Jenefer-Monroe** [comment]
Can you please share a screenshot like this so we can see what solutions you have installed in this environment? Be sure to expand the indicated columns so that we can see full names
![image](https://user-images.githubusercontent.com/89584709/221184839-4ed24d69-24ac-426e-a909-889e3b230166.png)

**Comment by dcurt1977** [comment]
These are the solutions installed in my environment.

![image](https://github.com/microsoft/coe-starter-kit/assets/123478097/77deb634-925b-41e2-a67f-6999006b7cd8)

**Comment by Jenefer-Monroe** [comment]
Can you delete if you try outside of the context of the solution like this?
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/4ed90cc2-56fe-487b-8fe7-4c97b6e977ca)

**Comment by dcurt1977** [comment]
Delete the highlighted connector?

![image](https://github.com/microsoft/coe-starter-kit/assets/123478097/e9e59d80-1af5-4b4b-b895-d5d05626cbfe)

**Comment by Jenefer-Monroe** [comment]
correct

**Comment by LizethRegules** [comment]
Hello, I´m facing the same issue here. 
I tried deleting the connector, but wasn´t able to. 
![image](https://github.com/microsoft/coe-starter-kit/assets/100951593/e211dbc6-6803-4038-bac2-8e229a45caa6)

And the connection reference it depends on, no longer exists in my environment, as I deleted the audit logs solution. 
![image](https://github.com/microsoft/coe-starter-kit/assets/100951593/e5656f5e-ddfc-4cb4-baa9-d5ed6a4b8fdc)

So, I'm not sure why it won't allow me to delete the connector and perform the update.
